doc/devguide.md

```diff
@@ -102,3 +102,52 @@ options
  -v, --image-version  Docker Image Version.
  -t, --template       Yaml file template for manifest definition.
 ```
+## Attach the VSCode Debugger to EdgeAgent
```

Comment:
Are there any pre-requisites? For example- IoT Edge Daemon should be already installed and running?

Reply:
There are no prerequisites other than having Edge Agent running and VSCode installed, which are implied by the steps listed below. 

Having IoT Edge Daemon running may also be an implicit pre-requisite because the daemon is probably needed in order to run Edge Agent.  However, if a user is able to somehow launch Edge Agent without the daemon, the steps below should still work. For that reason I think we shouldn't list it explicitly as a prerequisite.